src/steps/access.ts

```diff
@@ -107,32 +127,68 @@ export const accessSteps: IntegrationStep<IntegrationConfig>[] = [
     name: 'Fetch Users',
     entities: [
       {
-        resourceName: 'Account',
-        _type: 'acme_account',
+        resourceName: 'atSpoke Account',
+        _type: 'at_spoke_account',
         _class: 'Account',
       },
+      {
+        resourceName: 'atSpoke User',
+        _type: 'at_spoke_user',
+        _class: 'User',
+      },
     ],
     relationships: [
       {
-        _type: 'acme_account_has_user',
+        _type: 'at_spoke_account_has_user',
         _class: RelationshipClass.HAS,
-        sourceType: 'acme_account',
-        targetType: 'acme_user',
+        sourceType: 'at_spoke_account',
+        targetType: 'at_spoke_user',
       },
+    ],
+    dependsOn: ['fetch-account'],
+    executionHandler: fetchUsers,
+  },
+  {
+    id: 'fetch-groups',
+    name: 'Fetch UserGroups',
```

Comment:
I *think* these names tend to use the terminology of the provider, so that in the job logs presented to the users, they see that the atSpoke Teams have been processed.
```suggestion
    id: 'fetch-teams',
    name: 'Fetch Teams',
```

Reply:
True. Done.

Reply:
Agree. Done.